projects/module-1/ideabox-group.md

```diff
@@ -2,326 +2,235 @@
 
-As a user,
-- When I click the "Add" icon on an idea card,
-- A form to add a comment appears on that idea card
+![iteration 1](./assets/ideaBox/iteration1.png)
 
-As a user,
-- When I open the comment form on a card, type something in, and click "Add Comment",
-- The text typed in is now a comment attached to this card
+- When a user fills out both inputs and clicks "Save": 
+  - a new idea should be added to the data model.
+  - a card should appear on the screen to match the comp above. The card should show the title and body the user entered into the form.
+  - the form inputs should clear out to be ready for the next idea.
 
-As a user,
-- When I open the comment form on a card, type something in, and click "Add Comment",
-- The "Comment" input field clears out and is ready to accept another comment
+- When a user clicks "Save" with an empty Title or Body input field:
+  - the "Save" button is disabled and unable to be clicked. It's clear to the user when the button is disabled because it is a lighter color and the cursor is not a pointer when they hover over it.
+  - a new idea card should not appear.
 
-As a user,
-- When I open the comment form on am idea card,
-- When the "Comment" input field is empty,
-- I should notice that the "Add Comment" button is disabled because it is a lighter color and the cursor is not a pointer when I hover over it
+- Notes:
+  - You can choose what the layout looks like if more then 3 cards are added.
+  - You can choose how to handle titles and bodies that are longer than what fits on the card. There are lots of ways to handle this.
+</section>
 
-As a user,
-- If I have implemented localStorage,
-- When I comment on an idea card, then refresh the page,
-- That comment is still on the idea card
+<section class="answer">
+### Iteration 2 - Deleting Ideas
 
----
+![iteration 2](./assets/ideaBox/iteration2.png)
 
-## Notes on Project Feedback
+- A delete button should be visible on all idea cards now that match the comp above.
+- When a user clicks the delete button:
+  - that idea should be removed from the data model.
+  - that idea card should be removed from the page.
+</section>
 
-The rubric below serves as a guide for both **students** and **instructors**. 
+<section class="answer">
+### Iteration 3 - Favoriting Ideas
 
-**Students:**
-- It should be used as a reference throughout the project to keep on track and guide learning.
-- It should also be used by students to self-assess their work.
+![iteration 3](./assets/ideaBox/iteration3.png)
 
-**Instructors:**
-- It should be used to evaluate the project at its final due date/time.
-- It should be a guide to provide relevant feedback to students so they can grow and improve in the areas that need deeper understanding.
+- A star button should be visible on all idea cards now that match the comp above.
+- When a user clicks the star button:
+  - the data model should be updated. (You may want to add a new property to your idea objects)
```

Comment:
Is this too specific? Could we say something like "the data model should be updated such that you can tell which ideas have been starred" so that we're not quite saying exactly how to do it.

Reply:
fixed